src/mavoscript.js

```diff
@@ -23,14 +23,17 @@ var _ = Mavo.Script = {
 	},
 
 	binaryOperation: function(a, b, o = {}) {
-		var result;
+		var result = [];
```

Comment:
This only applies to the conditional below, so it's confusing to set it to an array here.